src/monster.h

```diff
@@ -384,6 +385,9 @@ class monster : public Creature
         void make_ally( const monster &z );
         // Add an item to inventory
         void add_item( const item &it );
+        // check mech power levels and modify it.
+        bool mod_mech_power( int amt );
```

Comment:
Calling this **mod**_tech_power is a bit confusing as it only allows to consume power, it does not allow to add power. 

Other *mod* functions (e.g. `player::mod_stat`) allow positive as well as negative values.